src/load/kialo.ts

```diff
@@ -0,0 +1,147 @@
+import * as model from "../model/index.js";
+import assert from "assert";
```

Comment:
why do we need a new dependency here? i would like to limit the dependencies to the bare minimum if possible